src/App.jsx

```diff
@@ -1,19 +1,25 @@
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import './App.css';
-import { LoginPage } from './components/LoginPage';
+import { LoginPage } from './components/login-page/LoginPage';
 import { HomePage } from './components/HomePage';
-// import CssBaseline from '@mui/material/CssBaseline';
+import AuthProvider from './components/login-page/AuthProvider';
+import PrivateRoute from './components/login-page/PrivateRoute';
 
 function App() {
     return (
-        <BrowserRouter>
-            {/* <div style={{ height: 200, width: 200, background: 'blue' }}></div> */}
-            <Routes>
-                <Route path="/login/*" Component={LoginPage} />
-                <Route path="/home-page" Component={HomePage} />
-                <Route path="/*" Component={LoginPage} />
-            </Routes>
-        </BrowserRouter>
+        <div>
+            <BrowserRouter>
+                <AuthProvider>
+                    <Routes>
+                        <Route path="/login/*" Component={LoginPage} />
```

Comment:
A suggestion for how we could improve routing.

instead of showing the Login Page for nearly everything (except the /home-page route), we could set it up something like this:
 <Route path="/login" Component={LoginPage} />
 <Route path="/home-page" Component={HomePage} /> - this one still inside a private route
 <Route path="/*" Component={NotFoundPage} />

and we can have a nice "not found page" with a button that sends you either to login or home